package/fedora/freecad.spec

```diff
@@ -274,12 +271,16 @@ popd
 
 # Remove obsolete Start_Page.html
 rm -f %{buildroot}%{_docdir}/%{name}/Start_Page.html
-# Belongs in %%license not %%doc
-rm -f %{buildroot}%{_docdir}/freecad/ThirdPartyLibraries.html
+# Keep this until transition from %%doc to %%license is done
+# and the tab is removed in UI
+# %{buildroot}%{_docdir}/freecad/ThirdPartyLibraries.html
```

Comment:
what transition are we talking about here? Are there plans to remove that tab from the UI? Might be a good idea to add a link to whatever relevant discussion there is

Reply:
You are right, a link would be good. I will try to find it.

Reply:
Mostly it comes from the old comment in the spec file where the page was not even packaged, which made the tab blank failing to find the page (so that is what I have fixed).

I see some comments in different threads about the redundancy. There is the section in "About > Licence" about third part licences pointing to the wiki. And each entry under "About > Libraries", which cmake preps, have a link to the licence it is released under. Maybe this transition is not happening? My summary might be misleading. @chennes is this About tabs redundancy something you know something about?

Reply:
the comment was 6 years old so I don't think any transition is in the works

Reply:
OK, so I remove it then.